### PR TITLE
Feat: Opinion and two fixtures

### PR DESCRIPTION
**Pull request by aronshamash**
- Fixtures we need in order to develop the Opinion and Two related articles
- Uploaded head and shoulders shot to s3

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/d37f3599537ad709333374375e596cd7d2af4d90/index.html

File: packages/article/related-articles/fixtures/opinionandtwo/1-article.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
+{
```
Comment (colincclark) [inline packages/article/related-articles/fixtures/opinionandtwo/1-article.json]: can we rename the lead fixtures to be inline with these?

Comment (aronshamash) [reply packages/article/related-articles/fixtures/opinionandtwo/1-article.json]: you mean use `"main"` ?

Comment (colincclark) [reply packages/article/related-articles/fixtures/opinionandtwo/1-article.json]: nope, file names

Comment (aronshamash) [reply packages/article/related-articles/fixtures/opinionandtwo/1-article.json]: updated

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/817fc8ea2aa94b247698b9a8fa1bead6dd2d2708/index.html

File: packages/article/__tests__/related-articles-shared.js [hunk 2]
```diff
@@ -6,12 +6,12 @@ import mockDate from "mockdate";
 
 import RelatedArticles from "../related-articles/related-articles";
 
-import standardSingleRelatedArticleFixture from "../related-articles/fixtures/standard/single-related-article.json";
-import standardTwoRelatedArticlesFixture from "../related-articles/fixtures/standard/two-related-articles.json";
-import standardThreeRelatedArticlesFixture from "../related-articles/fixtures/standard/three-related-articles.json";
-import leadAndTwoLeadOnlyRelatedArticleFixture from "../related-articles/fixtures/leadandtwo/lead-related-article.json";
-import leadAndTwoLeadAndSupportRelatedArticlesFixture from "../related-articles/fixtures/leadandtwo/lead-and-support-related-articles.json";
-import leadAndTwoLeadAndTwoSupportsRelatedArticlesFixture from "../related-articles/fixtures/leadandtwo/lead-and-two-supports-related-articles.json";
+import standard1ArticleFixture from "../related-articles/fixtures/standard/1-article.json";
```
Comment (colincclark) [inline packages/article/__tests__/related-articles-shared.js]: 👍 much better

File: packages/article/related-articles/fixtures/opinionandtwo/1-article.json (hunk 1 above)
Comment (colincclark) [reply packages/article/related-articles/fixtures/opinionandtwo/1-article.json]: thanks for doing that, much better

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/4ad2d5002f5b6431a62cfeb5684e48544af8a9f2/index.html

**Pull request merged**